src/styles.module.css

```diff
@@ -0,0 +1,150 @@
+:root {
+  --color-background: #F8F8F8;
+  --color-border: #EFF0F1;
+  --color-border-focus: #007AFF;
+
+  --color-card-title: #000;
+  --color-card-collection: #707684;
+
+  --border: 1px solid var(--color-border);
+}
+
+.wrapper {
+  margin: 1em auto;
+  display: flex;
+  flex-direction: row;
+
+  align-items: stretch;
+  padding: 20px 0;
+  gap: 30px;
+}
+
+.form {
+  width: 100%;
+
+  &-param:not(:last-child) {
+    margin-bottom: 16px;
+  }
+
+  &-label {
+    display: block;
+    font-weight: bold;
+    font-size: 14px;
+    padding-bottom: 6px;
+  }
+
+  &-input {
+    width: 100%;
+    padding: 8px 12px;
+    border-radius: 4px;
+    border: var(--border);
+    background-color: var(--color-background);
+    font-size: 14px;
+    line-height: 1.4;
+    outline: none;
+
+    &:focus {
+      border-color: var(--color-border-focus);
+    }
+  }
+
+  &-select {
+    width: 100%;
+    padding: 8px 12px;
+    border-radius: 4px;
+    border: var(--border);
+    background-color: var(--color-background);
+    font-size: 14px;
+    line-height: 1.4;
+    outline: none;
+
+
+    background-repeat: no-repeat;
+    background-position: right 10px bottom 50%;
+
+    cursor: pointer;
+
+    appearance: none;
+    -moz-appearance: none;
+    -webkit-appearance: none;
+
+    &:focus {
+      border-color: var(--color-border-focus);
+    }
+  }
+
+  &-button {
+    width: 100%;
+    padding: 8px 12px;
+    border-radius: 4px;
+    border: var(--border);
+    background-color: #fff;
+    font-size: 14px;
+    line-height: 1.4;
+    color: #333;
+    outline: none;
+
+    &:hover {
+      cursor: pointer;
+      background-color: rgba(0, 0, 0, 0.05);
+    }
+
+    &:active {
+      background-color: rgba(0, 0, 0, 0.1);
+    }
+
+    &:focus {
+      border-color: var(--color-border-focus);
+    }
+  }
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: column;
+  padding: 40px;
+
+  width: 230px;
+
+  border-radius: 16px;
+  border: var(--border);
+  box-shadow: 0px 2px 3px rgba(0, 0, 0, 0.03);
+
+  background: #FFFFFF;
+}
+
+.media {
+  border-radius: 16px;
+  overflow: hidden;
+
+  width: 100%;
+
+  background-color: #f0f0f0;
+
+  &::-webkit-scrollbar {
```

Comment:
Why do you need to disable a scrollbar here? Also, it wouldn't work for Mozilla.